src/components/AboutPage/Journey/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+
+const Journey: React.FC = () => {
+  const timelineData = [
+    {
+      year: "2005",
+      description:
+        "In 2005, Damien acquired her first camera, igniting her passion for photography. She started experimenting with landscapes and candid shots, marking the beginning of her visual storytelling journey.",
+    },
+    {
+      year: "2010",
+      description:
+        "Damien pursued a degree in Fine Arts with a focus on photography, refining her technical skills and artistic sensibilities.",
+    },
+    {
+      year: "2012",
+      description:
+        "In 2012, Damien held her first solo exhibition, showcasing her unique perspective on nature and people through her lens.",
+    },
+    {
+      year: "2015",
+      description:
+        "Damien officially launched her photography business in 2015, offering portrait, event, and commercial photography services.",
+    },
+    {
+      year: "2017",
+      description:
+        "Damien's work took her beyond the USA, capturing breathtaking scenes and cultures from around the world.",
+    },
+    {
+      year: "2020",
+      description:
+        "In 2020, Damien's dedication and creativity earned her several prestigious photography awards, solidifying her position as a visionary photographer.",
+    },
+  ];
```

Comment:
please move it outside the component
otherwise you will trigger unnecessary rerenders